debezium-connector-sqlserver/src/test/java/io/debezium/connector/sqlserver/SqlServerConnectorConfigTest.java

```diff
@@ -86,6 +86,15 @@ public void hostnameAndInstanceAndPortConnectionUrl() {
         assertEquals(connectionUrl(connectorConfig), "jdbc:sqlserver://${hostname}\\instance:${port}");
     }
 
+    @Test
+    public void validQueryFetchSize() throws Exception {
+        final SqlServerConnectorConfig connectorConfig = new SqlServerConnectorConfig(
```

Comment:
Please check `SqlServerConnection`. It uses a different code to obtain the fetch size.
Could you please rewrite the test in a way we can be sure the proper value is used or modify `SqlServerConnection` to use `getQueryFetchSize` method?

Reply:
Doesn't it fetch the same value though from JdbcConfig?

Reply:
The code in question is
```
this.queryFetchSize = config().getInteger(CommonConnectorConfig.QUERY_FETCH_SIZE);
```

So IMHO it will use the global default value not the one from SQL Server